src/pet.js

```diff
@@ -0,0 +1,55 @@
+const MAXIMUM_FITNESS = 10;
```

Comment:
Is there any reason why the have child and adopt methods werent included?
Also some error handling would be useful